core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_sidebar_directive.html

```diff
@@ -112,6 +112,9 @@
   simple-editor-sidebar .highlighted {
     background-color: #c5e1a5;
   }
+  simple-editor-sidebar .error {
```

Comment:
@SudhanvaMG Probably we're not using font-awesome (probably not 100% sure). Most icons at this point are thru SVG or Unicode symbol. Check this out, http://www.fileformat.info/info/unicode/char/26a0/index.htm